x-pack/plugins/monitoring/public/directives/cluster/listing/index.js

```diff
@@ -283,6 +388,9 @@ uiModule.directive('monitoringClusterListing', ($injector) => {
       const globalState = $injector.get('globalState');
       const kbnUrl = $injector.get('kbnUrl');
       const showLicenseExpiration = $injector.get('showLicenseExpiration');
+      const filterClusterPlaceholder = i18n('xpack.monitoring.cluster.listing.filterClustersPlaceholder',
```

Comment:
const filterClusterPlaceholder - >  const filterClustersPlaceholder
maybe add letter 's' in word clusters